src/vma/dev/net_device_table_mgr.cpp

```diff
@@ -323,10 +333,119 @@ void net_device_table_mgr::verify_ipoib_mode(struct ifaddrs* ifa)
 		vlog_printf(VLOG_WARNING,"This option in no longer needed in this version\n");
 		vlog_printf(VLOG_WARNING,"Please refer to Release Notes for more information\n");
 		vlog_printf(VLOG_WARNING,"************************************************************************\n");
+		return false;
 	}
 	else {
 		ndtm_logdbg("verified interface '%s' is running with umcast disabled", ifa->ifa_name);
 	}
+	return true;
+}
+
+bool net_device_table_mgr::check_ib_device_supported(const char* ifname)
+{
+	if (check_device_exist(ifname, MLX5_DRIVER_PATH)) {
+		vlog_printf(VLOG_WARNING,"************************************************************************\n");
+		vlog_printf(VLOG_WARNING,"Flow Steering of IPoIB traffic is not supported on mlx5 drivers\n");
```

Comment:
need to change WARNING to DEBUG.
there is nothing the user can do regarding this WARNING, he can't change any configuration to make it work.
need to quietly disable offload on this interface and log to DEBUG (all 3 lines are good for DEBUG, can remove the "*******" lines)